src/CoreShop/Bundle/ShippingBundle/Resources/translations/admin.de.yml

```diff
@@ -30,4 +29,6 @@ coreshop_condition_dimension: 'Dimension'
 coreshop_action_additionAmount: 'Aufschlag Menge'
 coreshop_action_additionPercent: 'Aufschlag Prozent'
 coreshop_shipping: 'Versand'
-coreshop_condition_amount_gross: 'Nutze Bruttowert'
+coreshop_shipping_tax_strategy: 'Steuerberechnung'
```

Comment:
please don't provide german translations, they'll get overwritten by crowdin anway

Reply:
shall i just delete the german translations?

Reply:
yes pls